### ISSUE TITLE
Pagination buttons have no associated label

### ISSUE THREAD

**Issue by jrpomeroy** [carbon-design-system/carbon#7261]
## Environment

> Operating system

OSX

> Browser

Latest Chrome

> Automated testing tool and ruleset

Accessibility Checker tool Chrome addon

> Assistive technology used to verify

## Detailed description

> What version of the Carbon Design System are you using?

carbon-components-react@7.23.2

> What did you expect to happen?

No violations in the tool

> What happened instead?

`Form control element <button> has no associated label`, on both pagination buttons

> What WCAG 2.1 checkpoint does the issue violate?

[IBM 3.3.2 Labels or Instructions](https://www.ibm.com/able/guidelines/ci162/labels_or_instructions.html)
[WCAG 2.1 technique H44](https://www.w3.org/TR/WCAG20-TECHS/H44)
[WCAG 2.1 technique ARIA16](https://www.w3.org/TR/WCAG20-TECHS/ARIA16.html)

## Steps to reproduce the issue

Go to https://react.carbondesignsystem.com/?path=/story/pagination--pagination and run the Accessibility Checker tool.

> Please create a reduced test case in CodeSandbox

Run Accessibility Checker tool on this page to reproduce:
https://react.carbondesignsystem.com/?path=/story/pagination--pagination

**Comment by emyarod** [comment]
should be resolved in https://github.com/carbon-design-system/carbon/pull/7253

**Comment by alon24** [comment]
Is this resolved now? or do you have an eta?

**Comment by emyarod** [comment]
this should be resolved now, are you still seeing the issue?

**Comment by dakahn** [comment]
closing this as resolved. we can reopen if that's not the case
